.github/renovate.json

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": ["github>balena-io/renovate-config"]
+  "extends": ["github>balena-io/renovate-config"],
+  "commitBody": "Update {{depName}}\n\nChange-type: patch"
```

Comment:
TBH we probably don't even need to set the `commitBody` here, since the defaults from balena-io should be fine.

Up to you, looks good otherwise.

Reply:
Well, then the inheritance isn't working: https://github.com/balena-os/renovate-config/pull/49/commits/1f6ffd289953094bbb912d1557f8c6482746ec10

Reply:
Huh, we should figure that out first then I think.